src/aura/STG_CMP_System/STG_CMP_System.cmp

```diff
@@ -48,8 +48,8 @@
                         </div>
 
                          <div class="slds-col slds-align-middle slds-gutters">
-                        	<h2 class="slds-text-heading_small">{!$Label.c.stgSuccessAdminAccountNameRefresh}</h2>
-                       	</div>
+                            <h2 class="slds-text-heading_small">{!$Label.c.stgSuccessAdminAccountNameRefresh}</h2>
```

Comment:
Spacing looks a little odd here.

Reply:
I'll take a look, will have to wait until tomorrow though.